lens-labels/lens-labels.cabal

```diff
@@ -0,0 +1,33 @@
+name:                lens-labels
+version:             0.1.0.0
+synopsis:            Integration of lenses with OverloadedLabels.
+description:         Provides a framework to integrate lenses with GHC's
+                     OverloadedLabels extension.
+homepage:            https://github.com/google/proto-lens
+license:             BSD3
+license-file:        LICENSE
+author:              Judah Jacobson
+maintainer:          judahjacobson+protolens@google.com
+copyright:           Google Inc.
+category:            Data
+build-type:          Simple
+cabal-version:       >=1.10
+
+library
+  exposed-modules:     Lens.Labels
+  build-depends:       base >= 4.8 && < 4.10
+                     , ghc-prim >=0.4 && <0.6
+  default-extensions:
+        CPP
```

Comment:
Would it be better to stick these into the .hs on the theory that they are the property of the source file (also more precise when different files need different extensions)?